apps/frontpage/components/docs/sidebar/sidebar.tsx

```diff
@@ -21,36 +27,39 @@ export const Sidebar: FC<SidebarProps> = ({ children }) => {
         <ScrollAreaPrimitive.Viewport className="h-full w-full rounded-[inherit]">
           <div className="py-12 pl-1 pr-4">
             <nav className="flex flex-col gap-1.5 text-sm font-medium">
-              <Link
-                className={cn(
-                  'flex items-center gap-3 hover:text-blue-500 transition-colors px-2 h-8',
-                  pathname === '/docs' && 'text-blue-500',
-                )}
+              <Line
                 href="/docs"
-              >
-                <DocsIcon />
-                Documentation
-              </Link>
-              <Link
-                className={cn(
-                  'flex items-center gap-3 hover:text-blue-500 transition-colors px-2 h-8',
-                  pathname === '/tutorials' && 'text-blue-500',
-                )}
-                href="#"
-              >
-                <TutorialsIcon />
-                Tutorials
-              </Link>
-              <Link
-                className={cn(
-                  'flex items-center gap-3 hover:text-blue-500 transition-colors px-2 h-8',
-                  pathname.startsWith('/releases') && 'text-blue-500',
-                )}
+                icon={<DocsIcon />}
+                isActive={
+                  pathname.startsWith('/docs') &&
+                  !pathname.startsWith('/docs/api')
+                }
+                label="Documentation"
+              />
+              <Line
+                href="/docs/api"
+                icon={<APIIcon />}
+                isActive={pathname.startsWith('/docs/api')}
+                label="API"
+              />
+              <Line
+                href="/#"
+                icon={<TutorialsIcon />}
+                isActive={pathname === '/tutorials'}
+                label="Tutorials"
+              />
+              <Line
+                href="/integrations"
+                icon={<IntegrationsIcon />}
+                isActive={false}
+                label="Integrations"
```

Comment:
We changed from `Integrations` to `Addons` in the header nav. I would think we'd want this to match?

Reply:
Good catch. I updated the label.